### DISCUSSION TITLE
404 File not Found

### DISCUSSION THREAD

**Discussion by FredesVirginia** [community/community#73344] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hola. Estaba trabajando en mi proyecto desde Codepens, y cuando hice un git push me decia repostorio no encontrado.
Luego entre a mis repositorios de git y la mayoria me salia el mismo error, que no se encontraban.
Alguien me puede ayudar, a recuperar mis repos? No se que puede haber pasado.

**Comment by queenofcorgis** [comment]
Hi @FredesVirginia , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
